### PR TITLE
WFLY-3918 Cannot disable ConsoleRedirectService

### PR DESCRIPTION
**Pull request by ctomc**

### CODE REVIEW

File: undertow/src/main/java/org/wildfly/extension/undertow/HostDefinition.java [hunk 1]
```diff
@@ -89,13 +88,17 @@ public void marshallAsAttribute(AttributeDefinition attribute, ModelNode resourc
...
-    .setDefaultValue(new ModelNode(404))
-    .build();
+            .setFlags(AttributeAccess.Flag.RESTART_RESOURCE_SERVICES)
+            .setValidator(new IntRangeValidator(400, 599, true, true))
+            .setDefaultValue(new ModelNode(404))
+            .build();
+    static final SimpleAttributeDefinition DISABLE_CONSOLE_REDIRECT = new SimpleAttributeDefinitionBuilder("disable-console-redirect", ModelType.BOOLEAN, true)
+            .setFlags(AttributeAccess.Flag.RESTART_RESOURCE_SERVICES)
+            .setDefaultValue(new ModelNode(false))
+            .build();
```
Comment (bstansberry) [inline undertow/src/main/java/org/wildfly/extension/undertow/HostDefinition.java]: Both this and DEFAULT_RESPONSE_CODE should allow expressions.

Comment (ctomc) [inline undertow/src/main/java/org/wildfly/extension/undertow/HostDefinition.java]: indeed they should

**Comment by bstansberry** [comment]
@stuartwdouglas please approve. looks fine to me.

**Comment by ctomc** [comment]
rebased

**Comment by stuartwdouglas** [comment]
Looks fine to m

**Pull request merged**